otter/test/rest/test_groups.py

```diff
@@ -833,26 +865,33 @@ def test_view_manifest_with_webhooks(self):
                 }
             ]
         ]
-        webhooks_internal_links = [
-            [[{"href": '/v1.0/11111/groups/one/policies/5/webhooks/3/', "rel": "self"},
+        webhooks_links = [
```

Comment:
This seems to change the test - it now declares `webhooks_links`, and then overwrites them with another copy.

This test is intended to test that the manifest lists all the webhooks for the policies (which was declared in `webhook_internal_links`), but also list a link to the webhooks resource for that policy (which was `webhooks_link`).